.changeset/eighty-buttons-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-examples': minor
```

Comment:
I have modified spelling in other examples, but that's not worth mentioning in changelog.